dmtestutils/login.py

```diff
@@ -0,0 +1,39 @@
+from flask import Blueprint
+from dmutils.user import User
+from flask_login import login_user
+
+login_for_tests = Blueprint('login_for_tests', __name__)
+
+
+USERS = {
+    '123': {
+        'id': 123,
+        'supplier_name': 'Supplier Name',
+        'name': 'Name',
+        'emailAddress': 'email@email.com',
+        'role': 'supplier',
+        'supplierId': 1234,
+    },
+    '234': {
+        'id': 234,
+        'name': 'Buyer',
+        'emailAddress': 'buyer@email.com',
+        'role': 'buyer',
+    },
+}
+
+
+@login_for_tests.route('/auto-login')
+def auto_login():
```

Comment:
Could/should this be `/auto-supplier-login` and `auto_supplier_login`?

(I'm not going to say we should have a version for each user role right now... but in an ideal world we would I think)

Reply:
Yeah I think it should really. That will mean slightly more major changes in order to bring it into the various frontends, tho. No time like the present eh...